transformer_lens/loading_from_pretrained.py

```diff
@@ -1652,20 +1675,30 @@ def convert_llama_weights(llama, cfg: HookedTransformerConfig):
         W_K = llama.model.layers[l].self_attn.k_proj.weight
         W_V = llama.model.layers[l].self_attn.v_proj.weight
         W_Q = einops.rearrange(W_Q, "(n h) m->n m h", n=cfg.n_heads)
-        W_K = einops.rearrange(W_K, "(n h) m->n m h", n=cfg.n_heads)
-        W_V = einops.rearrange(W_V, "(n h) m->n m h", n=cfg.n_heads)
+        W_K = einops.rearrange(
+            W_K, "(n h) m->n m h", n=cfg.n_key_value_heads if using_gqa else cfg.n_heads
+        )
+        W_V = einops.rearrange(
+            W_V, "(n h) m-n m h", n=cfg.n_key_value_heads if using_gqa else cfg.n_heads
```

Comment:
This einops pattern is missing the arrow, it should be "(n h) m->n m h"

Reply:
I'll just fix it with a PR. thanks for catching!